### DISCUSSION TITLE
preview server SDK 1.13, noticeType 16,  BT-71-Lot

### DISCUSSION THREAD

**Discussion by mDrozdAT** [OP-TED/eForms-SDK#1080] [Q&A] [answered]
Hi devs 

I have error msg (response from CVS server) 
'Reserved participation' (BT-71-Lot) is mandatory (in '16 – Contract notice – general directive, standard regime')

But i have it included in xml which is sent at CVS for validation 

Is this piece of XML correct ? If yes is everything allright at CVS server ? 

```XML
<cac:TenderingTerms>
    <cac:ProcurementLegislationDocumentReference>
      <cbc:ID>CrossBorderLaw</cbc:ID>
      <cbc:DocumentDescription languageID="DEU">some description</cbc:DocumentDescription>
    </cac:ProcurementLegislationDocumentReference>
    <cac:TendererQualificationRequest>
      <cac:SpecificTendererRequirement>
        <cbc:TendererRequirementTypeCode listName="exclusion-grounds-source">epo-procurement-document</cbc:TendererRequirementTypeCode>
      </cac:SpecificTendererRequirement>
    </cac:TendererQualificationRequest>
  </cac:TenderingTerms>
```

Thanks for answer

**Comment by bertrand-lorentz** [comment, marked as answer]
The XML snippet does not contain the field BT-71-Lot: this field uses the element `cbc:TendererRequirementTypeCode`, but with the attribute `listName="reserved-procurement"`

It seems you've put BT-806-Procedure instead, but this field should be at the procedure level, not under `cac:ProcurementProjectLot`

So the snippet should be something like:
```XML
<cac:TenderingTerms>
    <cac:ProcurementLegislationDocumentReference>
      <cbc:ID>CrossBorderLaw</cbc:ID>
      <cbc:DocumentDescription languageID="DEU">some description</cbc:DocumentDescription>
    </cac:ProcurementLegislationDocumentReference>
    <cac:TendererQualificationRequest>
      <cac:SpecificTendererRequirement>
        <cbc:TendererRequirementTypeCode listName="reserved-procurement">none</cbc:TendererRequirementTypeCode>
      </cac:SpecificTendererRequirement>
    </cac:TendererQualificationRequest>
  </cac:TenderingTerms>
```

**Reply by mDrozdAT** [reply]
ok. This section i have correct  (BT-806-Procedure) 
So this is different. 
Currently iam using 1.11 and there is no issue. 
in 1.13 conflict appear during creating XML 
BT-821-Lot is new Field and BT-771-Lot is more complex. 
Those 3 fields (BT-71/821/771-Lot) creating my issues and overwriting each others. 
Now i found it under the example. cn_24_FRA_comments.xml 
this is good example for me. Thanks for give me at right path (even its not exactly that :)
